src/sentry/testutils/helpers/datetime.py

```diff
@@ -10,3 +12,14 @@ def iso_format(date):
 
 def before_now(**kwargs):
     return datetime.utcnow() - timedelta(**kwargs)
+
+
+class MockClock(object):
+    """Returns a distinct, increasing timestamp each time it is called."""
```

Comment:
I poked around and couldn't find any existing test utils that do anything like this. Let me know if it's redundant.